jenkins/e2e-image/e2e-runner.sh

```diff
@@ -275,7 +275,7 @@ else
     # use JENKINS_PUBLISHED_VERSION, default to 'ci/latest', since that's
     # usually what we're testing.
     clean_binaries
-    fetch_published_version_tars "${JENKINS_PUBLISHED_VERSION:-ci/latest}"
+    fetch_published_version_tars "${JENKINS_PUBLISHED_VERSION:-ci/latest}" true
```

Comment:
it'd probably be less confusing if we just pulled `export CLUSTER_API_VERSION` out of `fetch_published_version_tars` entirely, since we only need it for one case, though I'm having trouble thinking of a good way to do that, since we depend on `$build_version`.

(we could have it set some variable that isn't used or exported? not sure if that's worse.)